enhancements/image-based-installer/image-based-installer.md

```diff
@@ -0,0 +1,219 @@
+acceleration of this adoption and the nature of existing Telecommunication
+infrastructure and processes drive the need to improve OpenShift provisioning
+speed at the Far Edge site and the simplicity of preparation and deployment of
+Far Edge clusters, at scale.
+
+The image-based installer provides users with such speed and simplicity, but it
+currently needs the [multicluster engine](https://docs.openshift.com/container-platform/4.15/architecture/mce-overview-ocp.html)
+and/or the [Image-based Install operator](https://github.com/openshift/image-based-install-operator)
+to generate the required installation and configuration artifacts. We would like
+to enable users to generate the latter intuitively and independently, using
+their own automation or even manual intervention to boot the host.
+
+### User Stories
+
+- As a user in a disconnected environment with no existing management cluster,
+  I want to deploy a single-node OpenShift cluster using a [seed image](https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/seed-image-generation.md)
+  and my own automation for provisioning.
+
+### Goals
+
+- Install clusters with single-node topology.
+- Install clusters in fully disconnected environments.
+- Perform reproducible cluster builds from configuration artifacts.
+- Require no machines in the cluster environment other than the one to be the
+  single node of the cluster.
+- Be agnostic to the tools used to provision machines, so that users can
+  leverage their own tooling and provisioning.
+
+### Non-Goals
+
+- Replace any other OpenShift installation method in any capacity.
+- Generate image formats other than ISO.
+- Automate booting of the ISO image on the machines.
+- Support installation configurations for cloud-based platforms.
+
+## Proposal
+
+A command-line tool will enable users to build a single custom RHCOS seed image
+in ISO format, containing the components needed to provision multiple
+single-node OpenShift clusters from that single ISO and multiple site
+configuration ISO images, one per cluster to be installed.
+
+The command-line tool will download the base RHCOS ISO, create an [Ignition](https://coreos.github.io/ignition/)
+file with generic configuration data (i.e. configuration that is going to be
+included in all clusters to be installed with that ISO) and produce an
+image-based installation ISO. The Ignition file will configure the live ISO such
+that once the machine is booted with the latter, it will install RHCOS to the
+installation disk, mount the installation disk, restore the single-node
+OpenShift from the [seed image](https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/seed-image-generation.md)
+and optionally precache all release container images under the
+`/var/lib/containers` directory. The command-line tool will also support
+generating a configuration ISO with all the site specific configuration data for
+the cluster to be installed provided as input.
```

Comment:
When possible, it would be good to have more details on how the second ISO works/is used.